packages/sit-onyx/src/components/OnyxSelectInput/OnyxSelectInput.vue

```diff
@@ -122,7 +122,7 @@ const blockTyping = (event: KeyboardEvent) => {
     ]"
     v-bind="rootAttrs"
   >
-    <OnyxFormElement v-bind="props" :error-messages="errorMessages">
+    <OnyxFormElement v-bind="props" :error-messages="errorMessages" @click="emit('click')">
```

Comment:
This would also emit the click event when clicking on the tooltip, message, label etc. which I am not sure if this should be the case.

Maybe we need to limit it to the input area only?

Reply:
But then it won't open when you click on the icon

Reply:
The icon is inside the `onyx-select-input__wrapper` so setting the click handler there should also work when clicking the icon :)